external/mdal/cmake/FindSQLite3.cmake

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2007-2009 LuaDist.
```

Comment:
this file should not be synced, it is already on cmake/ 

https://github.com/qgis/QGIS/blob/master/cmake/FindSqlite3.cmake